zcash_keys/src/keys.rs

```diff
@@ -654,6 +660,19 @@ impl ReceiverRequirements {
         Self::unsafe_new(Allow, Allow, Allow)
     };
 
+    /// Constructs a new unified address request that allows only shielded receivers
+    pub const SHIELDED: ReceiverRequirements = {
+        use ReceiverRequirement::*;
+        Self::unsafe_new(Allow, Allow, Omit)
+    };
+
+    /// Constructs a new unified address request that requires an Orchard receiver,
+    /// and no others.
+    pub const ORCHARD: ReceiverRequirements = {
```

Comment:
These now need to be documented as additions in the changelog, as `ReceiverRequirements` was added in `zcash_keys 0.8.0`.